### DISCUSSION TITLE
Errors in installing RACCOON.

### DISCUSSION THREAD

**Discussion by zuckarm** [idaholab/moose#27021] [Q&A General]
Hi all, i have followed the steps given on https://hugary1995.github.io/raccoon/getting_started/install.html
However, i am getting errors while executing the 'make' command:
![Screenshot from 2024-03-08 01-06-36](https://github.com/idaholab/moose/assets/162629698/d9e0e84c-236f-4d37-b638-a0427dddfe56)
Can someone please help out with this error?

**Comment by GiudGiud** [comment]
Hello

Your libmesh installation (you use conda to install I figure) is not up to date.
Update with `conda update moose-dev`

Guillaume
